src/components/AddressDetails/AddressDetails.tsx

```diff
@@ -28,46 +50,53 @@ const AddressDetails = ({ data, isLoading, updateCaptureOption }) => {
     }
   }, [data]);
 
-  const onAddressChange = (key, newValue) => {
-    setAddress({
-      ...address,
-      [key]: newValue
-    });
-  };
+  const onAddressChange =
```

Comment:
this will get recreated on each render. please wrap this function with `useCallback`

Reply:
Do you think there will be any benefit of using `useCallback` on such a simple function? I was told that `useCallback` also comes with a cost and should be used when the function is really complex and causes slowness because otherwise it does not give benefit but it decreases readability